rma/models/rma_order_line.py

```diff
@@ -98,12 +101,24 @@ def _get_rma_move_qty(self, states, direction="in"):
             product_obj = self.env["uom.uom"]
             qty = 0.0
             if direction == "in":
-                op = ops["="]
-            else:
-                op = ops["!="]
-            for move in rec.move_ids.filtered(
-                lambda m: m.state in states and op(m.location_id.usage, rec.type)
-            ):
+                moves = rec.move_ids.filtered(
+                    lambda m: m.state in states
+                    and (
+                        m.location_id.usage == "supplier"
+                        or m.location_id.usage == "customer"
+                    )
+                    and m.location_dest_id.usage == "internal"
+                )
+            elif direction == "out":
+                moves = rec.move_ids.filtered(
+                    lambda m: m.state in states
+                    and (
+                        m.location_dest_id.usage == "supplier"
+                        or m.location_dest_id.usage == "customer"
+                    )
+                    and m.location_id.usage == "internal"
```

Comment:
```suggestion
                    and (m.location_id.usage == "internal"
                       or m.location_dest_id.usage == "customer"
                    )
```

When the RMA is 'Dropship deliver to customer' it is not getting the moves correctly, because the location_id.usage is "supplier" but the location_dest_id.usage is "customer". and does not take into account this option.